packages/containers/src/HeaderBar/HeaderBar.component.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { cmfConnect } from '@talend/react-cmf';
+import { HeaderBar as PureHeaderBar } from '@talend/react-components';
+import { Action } from '../index';
+
+function HeaderBar(props) {
+	const renderers = {
+		Action,
+	};
+	return <PureHeaderBar renderers={renderers} {...props} />;
+}
+
+export default cmfConnect({})(HeaderBar);
```

Comment:
Please follow the pattern in place on the other containers : 
* a container file
* a connect file

Reply:
all other container don't use this pattern, because there is not always need for a container to manage some middleware, here it is the case no place for a container and the connect is only here for the sake of automatic view setting injection.